include/struct_decls/struct_0202CC84_decl.h

```diff
@@ -1,6 +1,6 @@
 #ifndef POKEPLATINUM_STRUCT_0202CC84_DECL_H
 #define POKEPLATINUM_STRUCT_0202CC84_DECL_H
 
-typedef struct UnkStruct_0202CC84_t UnkStruct_0202CC84;
+typedef struct ChatotCryData_t ChatotCryData;
```

Comment:
Can you merge this into the struct-definition header? In general, we are trying to get rid of opaque structs.